### PR TITLE
doc: minor fixes

### PR DESCRIPTION
**Pull request by drunkard**

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -604,9 +604,9 @@ Changing the Metadata Master Zone
 To promote a zone (for example, zone ``us-2`` in zonegroup ``us``) to metadata
 master, run the following commands on that zone::
 
-  $ radosgw-admin zone modify --rgw-zone=us-2 --master
```
Comment (tchaikov) [inline doc/radosgw/multisite.rst]: why do we need to re-indent these commands?

File: doc/radosgw/multisite.rst [hunk 2]
```diff
@@ -617,7 +617,7 @@ Failover and Disaster Recovery
 If the master zone should fail, failover to the secondary zone for
 disaster recovery.
 
-1. Make the secondary zone the master and default zone. For example:
+#. Make the secondary zone the master and default zone. For example:
```
Comment (tchaikov) [inline doc/radosgw/multisite.rst]: why is this a mistake?

Comment (drunkard) [inline doc/radosgw/multisite.rst]: We uses `#.` everywhere, and from `git log` we also fixed some `[0-9].`.

Comment (tchaikov) [inline doc/radosgw/multisite.rst]: i am not sure i am following you. so it's more about consistency. and how come this is related to `git log`?

File: doc/radosgw/multisite.rst (hunk 1 above)
Comment (drunkard) [inline doc/radosgw/multisite.rst]: All commands except this one uses 4 spaces as indention.

Comment (tchaikov) [inline doc/radosgw/multisite.rst]: so, again this is all about consistency. i'd lean towards keeping as it is. and change it when we happen to touch this area.

File: doc/radosgw/multisite.rst (hunk 2 above)
Comment (drunkard) [inline doc/radosgw/multisite.rst]: Yes, it's about document style consistency, either we fix it now, or fix when add/remove sections.

So let it be for now?

**Pull request merged**